src/docs/feature-flags.mdx

```diff
@@ -235,22 +247,22 @@ segments is that each segment is **OR**ed together, and each constraint is
 Properties are a preset dictionary that get sent to flagr by sentry, these depend
 on the organization or project being passed to `features.has`
 
-| Property                        | Value Types  | Notes                       |
-| :-----------------------------  | :----------- | :-------------------------- |
-| `organization_id`               | `int`        |                             |
-| `organization_slug`             | `string`     |                             |
-| `organization_is-early-adopter` | `bool`       | Users opt in from settings  |
-| `features`                      | `[string]`   | All active features for org |
-| `subscription_is-free`          | `bool`       |                             |
-| `subscription_is-trial-plan`    | `bool`       |                             |
-| `subscription_plan`             | `string`     |                             |
-| `project_slug`                  | `string`     | Only passed for projects    |
-| `project_id`                    | `int`        | Only passed for projects    |
-| `user_id`                       | `int`        |                             |
-| `user_domain`                   | `string`     |                             |
-| `user_is-superuser`             | `bool`       |                             |
-| `user_is-staff`                 | `bool`       |                             |
-| `internal_teams`                | `[string]`   | Only passed for Sentry      |
+| Property                        | Value Types  | Notes/Example config               |
+| :-----------------------------  | :----------- | :--------------------------------- |
+| `organization_id`               | `int`        |                                    |
+| `organization_slug`             | `string`     | [Example](#releasing-a-feature-to-a-subset-of-organizations) |
```

Comment:
ugh, not sure how to format the `.mdx` to make this less ugly :( now all the `|` are all over the place

Reply:
You could align the cell boundaries with the longest cell?

Reply:
😬 yeah but it gets really long, but I guess that's better than this. 👍